packages/forms/src/UIForm/fields/Code/Code.component.js

```diff
@@ -1,198 +1,170 @@
-				errorId={errorId}
-				errorMessage={errorMessage}
-				id={id}
-				isValid={isValid}
-				label={title}
-				labelProps={labelProps}
-				required={schema.required}
-				valueIsUpdating={valueIsUpdating}
+	return (
+		<FieldTemplate
+			description={description}
+			descriptionId={descriptionId}
+			errorId={errorId}
+			errorMessage={errorMessage}
+			id={id}
+			isValid={isValid}
+			label={title}
+			labelProps={labelProps}
+			required={schema.required}
+			valueIsUpdating={valueIsUpdating}
+		>
+			<div // eslint-disable-line jsx-a11y/no-static-element-interactions
+				id={id && `${id}-editor-container`}
+				onBlur={onBlur}
+				onKeyDown={onKeyDown}
+				ref={containerRef}
+				tabIndex="-1"
 			>
-				<div // eslint-disable-line jsx-a11y/no-static-element-interactions
-					id={id && `${id}-editor-container`}
-					onBlur={this.onBlur}
-					onKeyDown={this.onKeyDown}
-					ref={ref => {
-						this.ref = ref;
-					}}
-					tabIndex="-1"
-				>
-					<div id={instructionsId} className="sr-only">
-						{t('TF_CODE_ESCAPE', {
-							defaultValue: 'To focus out of the editor, press ESC key twice.',
-						})}
-					</div>
-					<AceEditor
-						key="ace"
-						className="tf-widget-code form-control"
-						editorProps={{ $blockScrolling: Infinity }} // https://github.com/securingsincity/react-ace/issues/29
-						focus={autoFocus}
-						name={`${id}_wrapper`}
-						mode={options && options.language}
-						onBlur={this.onFinish}
-						onLoad={this.onLoad}
-						onChange={this.onChange}
-						// disabled is not supported by ace use readonly
-						// https://github.com/ajaxorg/ace/issues/406
-						readOnly={readOnly || schema.disabled || valueIsUpdating}
-						setOptions={DEFAULT_SET_OPTIONS}
-						showGutter={false}
-						showPrintMargin={false}
-						theme="chrome"
-						value={value}
-						width="auto"
-						{...options}
-					/>
+				<div id={instructionsId} className="sr-only">
+					{t('TF_CODE_ESCAPE', {
+						defaultValue: 'To focus out of the editor, press ESC key twice.',
+					})}
 				</div>
-			</FieldTemplate>
-		);
-	}
-}
-if (process.env.NODE_ENV !== 'production') {
-	WrappedTextArea.propTypes = TextArea.propTypes;
-	Code.propTypes = {
-		id: PropTypes.string,
-		isValid: PropTypes.bool,
-		errorMessage: PropTypes.string,
-		onChange: PropTypes.func.isRequired,
-		onFinish: PropTypes.func.isRequired,
-		schema: PropTypes.shape({
-			autoFocus: PropTypes.bool,
-			description: PropTypes.string,
-			disabled: PropTypes.bool,
-			options: PropTypes.object,
-			readOnly: PropTypes.bool,
-			required: PropTypes.bool,
-			title: PropTypes.string,
-			labelProps: PropTypes.object,
-			type: PropTypes.string,
-		}),
-		t: PropTypes.func,
-		value: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
-		valueIsUpdating: PropTypes.bool,
-	};
+				<ImportLazy
+					skeleton={<CodeSkeleton />}
+					name="react-ace"
+					version="6.2.0"
+					varName="ReactAce"
+					path="/dist/react-ace.min.js"
```

Comment:
proposal: add this from modules.json at build time only (we don't want the 1k lines of json at runtime)